Assets/Tests/InputSystem/Assets/UITKTestScene.unity

```diff
@@ -0,0 +1,434 @@
+%YAML 1.1
```

Comment:
I opened the scene (in Unity 2021.2.0a9) and noticed the UIDocument was missing its PanelSettings. Maybe it's just a different version of Unity where the meta has changed (package vs trunk), but I just wanted to make sure this wasn't removed by accident.
![image](https://user-images.githubusercontent.com/59068862/111678006-31138700-87f6-11eb-8481-2680b5ce7593.png)

Also I had to add InputSystem to the PackageManager because the manifest.json file seemed to be .gitignore'd. Was that intentional?

Reply:
>I opened the scene (in Unity 2021.2.0a9) and noticed the UIDocument was missing its PanelSettings. Maybe it's just a different version of Unity where the meta has changed (package vs trunk), but I just wanted to make sure this wasn't removed by accident.

Indeed. Didn't realize something got added to the project elsewhere. Checked the file in. Reference should be fine now.

>Also I had to add InputSystem to the PackageManager because the manifest.json file seemed to be .gitignore'd. Was that intentional?

Hmm, manifest.json .gitignored? According to our .gitignores, shouldn't be happening.

The input system, however, doesn't need to be in the manifest. In this project, it's an embedded package.

Reply:
Think I finally checked in all missing files.